earthmover/package.py

```diff
@@ -166,7 +166,7 @@ class GitHubPackage(Package):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def install(self, packages_dir):
+    def install(self, packages_dir, git_auth_timeout):
```

Comment:
As mentioned above, there's another world where we set the default `git_auth_timeout` value to 60 here, or we should add it to the `Earthmover.config_defaults` object such that it is always defined in a project.